package.json

```diff
@@ -14,6 +14,7 @@
     "debug": "~2.1.1",
     "express": "~4.11.1",
     "express-session": "^1.10.3",
+    "fastclick": "^1.0.6",
```

Comment:
This seems to be an error actually. We shouldnt include fastclick on the browser